src/test/java/uk/gov/hmcts/reform/pip/data/management/service/filegeneration/nonstrategic/UtIacJudicialReviewLeedsDailyHearingListFileConverterTest.java

```diff
@@ -0,0 +1,274 @@
+
+    private final NonStrategicListFileConverter converter = new NonStrategicListFileConverter();
+
+    private JsonNode utIacJudicialReviewInputJson;
+
+    @BeforeAll
+    void setup() throws IOException {
+        try (InputStream inputStream = getClass()
+            .getResourceAsStream("/mocks/non-strategic/utIacJudicialReviewLeedsDailyHearingList.json")) {
+            String inputRaw = IOUtils.toString(inputStream, Charset.defaultCharset());
+            utIacJudicialReviewInputJson = new ObjectMapper().readTree(inputRaw);
+        }
+    }
+
+    @Test
+    void testUtIacJudicialReviewDailyHearingListFileConversionInEnglish() throws IOException {
+        Map<String, Object> languageResource;
+        try (InputStream languageFile = Thread.currentThread()
+            .getContextClassLoader()
+            .getResourceAsStream("templates/languages/en/non-strategic/utIacJrLeedsDailyHearingList.json")) {
+            languageResource = new ObjectMapper().readValue(
+                Objects.requireNonNull(languageFile).readAllBytes(), new TypeReference<>() {
+                });
+        }
+
+        Map<String, String> metadata = Map.of(CONTENT_DATE_METADATA, CONTENT_DATE,
+                                              PROVENANCE_METADATA, PROVENANCE,
+                                              LANGUAGE_METADATA, ENGLISH,
+                                              LIST_TYPE_METADATA, UT_IAC_JR_LEEDS_DAILY_HEARING_LIST.name(),
+                                              LAST_RECEIVED_DATE_METADATA, LAST_RECEIVED_DATE
+        );
+
+        String result = converter.convert(utIacJudicialReviewInputJson, metadata, languageResource);
+        Document document = Jsoup.parse(result);
+
+        SoftAssertions softly = new SoftAssertions();
+
+        softly.assertThat(document.title())
+            .as(TITLE_MESSAGE)
+            .isEqualTo("Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Leeds Daily Hearing List");
+
+        softly.assertThat(document.getElementById(HEADER_ELEMENT).text())
+            .as(HEADER_MESSAGE)
+            .isEqualTo("Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Leeds Daily Hearing List");
+
+        softly.assertThat(document.getElementById(LIST_DATE_ELEMENT).text())
+            .as(LIST_DATE_MESSAGE)
+            .isEqualTo(LIST_DATE_ENGLISH);
+
+        softly.assertThat(document.getElementById(LAST_UPDATED_DATE_ELEMENT).text())
+            .as(LAST_UPDATED_DATE_MESSAGE)
+            .isEqualTo("Last updated 20 January 2025 at 9:30am");
+
+        softly.assertThat(document.getElementsByClass(SUMMARY_TEXT_CLASS).get(0).text())
+            .as(IMPORTANT_INFORMATION_MESSAGE)
+            .isEqualTo("Important information");
+
+        softly.assertThat(document.getElementById(LIST_UPDATE_MESSAGE_ELEMENT).text())
+            .as(BODY_MESSAGE)
+            .contains("The following list is subject to change until 4:30pm");
```

Comment:
It will work but can you add full open justice wording to make it consistent with other lists.